mmpose/datasets/pipelines/bottom_up_transform.py

```diff
@@ -242,6 +242,90 @@ def __call__(self, joints):
         return visible_kpts
 
 
+class PAFGenerator:
+    """Generate part affinity fields.
+
+    Args:
+        output_res (int): Size of feature map.
```

Comment:
Why not use `output_size`

Reply:
res for resolution?

Reply:
It follows the JointsEncoder class. Maybe we can change all the 'output_res' to 'output_size'.